Examples/Examples.m

```diff
@@ -19,6 +19,7 @@ + (NSArray *)groups {
                     @{@"className": MBXExampleCameraAnimation, @"title": @"Camera animation"},
                     @{@"className": MBXExampleStudioStyle, @"title": @"Apply a style designed in Mapbox Studio"},
                     @{@"className": MBXExampleSimpleMapView, @"title": @"Simple map view"},
+                    @{@"className": MBXExampleLocationPrivacy, @"title": @"Location privacy iOS 14 updates"},
```

Comment:
Shall we update name to "Using Approximate Location in iOS 14"?

Reply:
yeah that makes more sense